WindowManager/app/src/main/java/com/example/windowmanagersample/WindowDemosActivity.kt

```diff
@@ -40,17 +42,15 @@ class WindowDemosActivity : AppCompatActivity() {
         val view = binding.root
         setContentView(view)
 
-        binding.backendRadioGroup.setOnCheckedChangeListener { _, checkedId ->
-            when (checkedId) {
-                R.id.device_default_radio_button ->
-                    selectedBackend = BACKEND_TYPE_DEVICE_DEFAULT
-                R.id.mid_fold_radio_button ->
-                    selectedBackend = BACKEND_TYPE_MID_SCREEN_FOLD
+        val radioGroup = binding.backendRadioGroup
+        radioGroup.setOnCheckedChangeListener { _, checkedId ->
```

Comment:
Is this just the overall better practice instead of doing it in one line?

Reply:
I'm using the same binding below, I can avoid a local val, but was saving me some typing below.